genie-agent/src/main/java/com/netflix/genie/agent/cli/Util.java

```diff
@@ -0,0 +1,66 @@
+/*
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package com.netflix.genie.agent.cli;
+
+import java.util.Arrays;
+
+/**
+ * CLI utility methods.
+ *
+ * @author mprimi
+ * @since 4.0.0
+ */
+public final class Util {
+
+    /**
+     * Bare double dash is a command-line option conventionally used to delimit options from arguments.
+     * See POSIX POSIX.1-2017 - Utility Conventions - Gudeline #10
+     * http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html
+     */
+    static final String BARE_DOUBLE_DASH = "--";
+    private static final String BARE_DOUBLE_DASH_REPLACEMENT = "-//-";
+
+    /**
+     * Hidden constructor.
+     */
+    private Util() {
+    }
+
+    /**
+     * Replace all "bare double dash" arguments in the input array.
+     * @param args the arguments array
```

Comment:
Does your auto config within intellij not put a `\n` between the function description and the first `@param` tag? I'm just curious cause I've seen some of @sumitnetflix code format like this too and I don't want to get those weird just formatting commits so we should settle on one or the other

Reply:
It doesn't in the template, but it does when i do a full file re-format of the file.
I'll fix this, and I think it does look better and we should stick with it where possible.

Reply:
Fixed